roles/identity-management/manage-rh-sso-identities/README.md

```diff
@@ -0,0 +1,47 @@
+Manage Red Hat Single Sign-on identities (users/groups)
+=======================================================
+
+An ansible role that manages Red Hat Single Sign-On identities - users and groups.
+
+
+Role Variables
+--------------
+
+| Variable | Description | Required | Defaults |
+|:--------:|:-----------:|:--------:|:--------:|
+|**rh_sso_host**|The hostname/ip used to conmect to for RH SSO management|yes|N/A|
+|**sso_admin_user**|The RH SSO admin user with proper permissions to administer identities|yes|N/A|
+|**sso_admin_pass**|The RH SSO admin password for the above mentioned admin user|yes|N/A|
+|**user_name**|The username used by the user to authenticate|yes|N/A|
```

Comment:
These are confusing here since it should be a list of users, and we should just reference the **list** of users one level up - just like we do here https://github.com/redhat-cop/infra-ansible/tree/master/roles/identity-management/manage-idm-identities#role-variables

Reply:
Great, I updated the file to reflect the correct format.